src/main/java/com/codeborne/selenide/impl/DownloadFileWithCdp.java

```diff
@@ -126,17 +126,16 @@ private File waitUntilDownloadsCompleted(Driver driver, FileFilter fileFilter,
   }
 
   private DevTools initDevTools(Driver driver) {
-    if (driver.browser().isChromium()) {
-      Optional<HasDevTools> cdpBrowser = cast(driver.getWebDriver(), HasDevTools.class);
-      if (cdpBrowser.isPresent()) {
-        DevTools devTools = cdpBrowser.get().getDevTools();
-        devTools.createSessionIfThereIsNotOne();
-        devTools.send(Page.enable());
-        return devTools;
-      }
+    Optional<HasDevTools> cdpBrowser = cast(driver.getWebDriver(), HasDevTools.class);
+    if (cdpBrowser.isPresent()) {
+      DevTools devTools = cdpBrowser.get().getDevTools();
```

Comment:
The problem is that Firefox doesn't support downloading files via CDP.
We need to exclude Firefox in this IF. @PetroOv

Reply:
I didn't know it( I returned isChromium check, but now we get browser name from capabilities of instance